.travis/update-cmake.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+
+# Make a directory for cmake binaries
+mkdir -p $HOME/cmake_install
```

Comment:
Is there a difference between this and $HOME/cmake-install defined in travis/install-deps.sh?

Reply:
This file was an attempt at a workaround to travis handling the user input request poorly. It's no longer used, this file has been deleted.